controllers/question.controller.js

```diff
@@ -0,0 +1,42 @@
+const utility = require('../lib/utility.lib');
+const { createQuestion, destroyQuestion } = require('../lib/question.lib');
+
+const {
+  filterValues, formatValues,
+} = utility;
+
+const postQuestion = async (req, res) => {
+  try {
+    const rawData = req.body;
+    const filteredValues = filterValues(rawData, ['question_text']);
+    const data = formatValues(filteredValues);
+
+    const question = await createQuestion({
+      question_text: data.question_text,
+      user_id: 1,
+    });
```

Comment:
Hardcoding user_id, I guess it's a mistake, it should be gotten from the `req` object injected in `authentication.middleware.js`

Reply:
Yes, I put it before setting up the middleware